nailgun/entities.py

```diff
@@ -3105,26 +3205,77 @@ def __init__(self, server_config=None, **kwargs):
         super(OSDefaultTemplate, self).__init__(server_config, **kwargs)
 
 
-class OverrideValue(Entity):
+class OverrideValue(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntityUpdateMixin):
     """A representation of a Override Value entity."""
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
-            'match': entity_fields.StringField(),
+            'match': entity_fields.StringField(required=True),
+            'value': entity_fields.StringField(required=True),
+            'smart_class_parameter': entity_fields.OneToOneField(
+                SmartClassParameters),
             'smart_variable': entity_fields.OneToOneField(SmartVariable),
-            'value': entity_fields.StringField(),
+            'use_puppet_default': entity_fields.BooleanField(),
         }
+        super(OverrideValue, self).__init__(server_config, **kwargs)
+        # Create an override value for a specific smart class parameter
+        if hasattr(self, 'smart_class_parameter'):
+            # pylint:disable=no-member
+            partial_path = self.smart_class_parameter. path('self')
```

Comment:
`partial_path = self.smart_class_parameter. path('self')`?
not the
`partial_path = self.smart_class_parameter.path('self')`?
How that typo even passed validation? Does `# pylint:disable=no-member` affect that? If yes, that is bad that we can miss such errors